src/goto-programs/goto_trace.h

```diff
@@ -117,6 +117,7 @@ class goto_trace_stept
   // for assume, assert, goto
   bool cond_value;
   exprt cond_expr;
+  exprt original_condition;
 
```

Comment:
The difference between `original_condition` and `cond_expr` is non-obvious, and does deserve a comment.
Do we really need both?

Reply:
I was tempted to just re-purpose `cond_expr`, but it turns out it is actually used by `goto-checker/bmc_util.cpp` (I believe that is the only use). `cond_expr` is the SSA-renamed expression, while we'd want to provide an expression that is meaningful to the user. So maybe adding a comment is the right approach?